### DISCUSSION TITLE
iOS: App Extension not working since latest mobile app version (2.14.2)

### DISCUSSION THREAD

**Discussion by crivchri** [dani-garcia/vaultwarden#2109] [Q&A]
### Subject of the issue
Since the latest version 2.14.2 of the iOS mobile application, Bitwarden is not showing up in the list of applications.

### Deployment environment

* vaultwarden version: v1.23.0

* Install method: Docker (Base: Debian)

* Clients used: iOS

* Reverse proxy and version: Synology

* MySQL/MariaDB or PostgreSQL version:  SQLite 3.35.4

* Other relevant details:
Also reported as bug to Bitwarden: https://github.com/bitwarden/mobile/issues/1635

### Steps to reproduce

1. Go to 'Settings'
2. Click on 'Application extension'
3. Click on 'Activate…'

### Expected behaviour

Bitwarden showing up in the list.

### Actual behaviour

Bitwarden showing not up in the list.

**Comment by BlackDex** [comment]
I'm not sure what this has to do with Vaultwarden actually.
Looks more like a client application issue, which are not maintained by this project.
